examples/ffmpeg/notes.md

```diff
@@ -0,0 +1,42 @@
+* dffml service dev create operations ffmpeg
+* write operations and definitions
+* change setup.py
+*
+    ```
+    pip install -e .
+
+    cat > /tmp/operations/ <<EOF
+    convert_to_gif
+    EOF
+
+    dffml dataflow create -config yaml $(cat /tmp/operations) > deploy/df/ffmpeg.yaml
+
+    cd ffmpeg
+    mkdir -p deploy/mc/http
+        ffmpeg.yaml
+        path: /ffmpeg
+        presentation: json
+        asynchronous: false
+
+    dffml service http server -insecure -mc-config deploy
+
+    ```
+
+* curl -s \
+  --header "Content-Type: application/json" \
+  --request POST \
+  --data '{"convert": [{"value":"input.mp4","definition":"input_file"},{"value":"output.gif","definition":"output_file"}]}' \
+  http://localhost:8080/ffmpeg
+
+  works!!!
```

Comment:
:D

Reply:
oh i didnt notice this snuck in!!

Reply:
All good, I assume this file is the start of what's going to end up in `docs/usage/deploy.rst`